src/components/Searchbar.js

```diff
@@ -0,0 +1,17 @@
+import "./Searchbar.css";
+export default function SearchBar({ onNameChange }) {
+  function handleOnNameFilter(event) {
+    event.preventDefault();
+    const filterNameValue = event.target.value;
+    onNameChange(filterNameValue);
+  }
```

Comment:
I recommend removing this handler function and adding another prop to this component with the functionality of holding a value for the input field. That way this component can be a controlled component and as flexible as possible.

Props for the input and this component
```
value={value}
onChange={onChange}
```